src/AccountLabel.jsx

```diff
@@ -58,54 +60,59 @@ class AccountLabelAux extends ReactiveComponent {
 	readyRender () {
 		let badges = this.state.badges.map((b, i) => (
```

Comment:
Try to use more meaningful variable names, b and i.

Idem in other files

Reply:
updated